apps/console/package.json

```diff
@@ -159,7 +159,6 @@
         "connect-history-api-fallback": "^2.0.0",
         "copy-webpack-plugin": "^12.0.2",
         "css-loader": "^1.0.0",
-        "eslint": "^7.20.0",
```

Comment:
Is this expected?

Reply:
Fixed with [d0f8fab](https://github.com/wso2/identity-apps/pull/6487/commits/d0f8fab615c67a855bbf44a0fb3bc7a77aae9c94)